server/build-osx.sh

```diff
@@ -199,7 +199,8 @@ _build_server_osx() {
     # Building 'System_stats' extention and bundled the required files with posgresql
     ssh $PG_SSH_OSX "cd $PG_PATH_OSX/server/source/system_stats.osx; PATH="$PG_STAGING/bin:$PATH" make USE_PGXS=1; PATH="$PG_STAGING/bin:$PATH" make install USE_PGXS=1" || _die "Failed to build System Status"
 
-    ssh $PG_SSH_OSX "cd $PG_PATH_OSX/server/source/system_stats.osx/; cp system_stats--*.sql system_stats.control $PG_STAGING/share/postgresql/extension; cp system_stats.so $PG_STAGING/lib/postgresql " || _die "Failed to bundle system_stats extension files"
+    #ssh $PG_SSH_OSX "cd $PG_PATH_OSX/server/source/system_stats.osx/; cp system_stats--*.sql system_stats.control $PG_STAGING/share/postgresql/extension; cp system_stats.so $PG_STAGING/lib/postgresql " || _die "Failed to bundle system_stats extension files"
+     ssh $PG_SSH_OSX "cd $PG_PATH_OSX/server/source/system_stats.osx/; cp system_stats--*.sql system_stats.control $PG_STAGING/share/postgresql/extension " || _die "Failed to bundle system_stats extension files"
```

Comment:
either .so or .dylib should be copied, which is missing here

Reply:
Adding that